infra/services/aiSearch-post.ps1

```diff
@@ -0,0 +1,181 @@
+param(
+    [string] $ResourceGroupName,
+    [string] $BaseName
+)
+
+$ErrorActionPreference = "Stop"
+
+. "$PSScriptRoot/../../eng/common/scripts/common.ps1"
+
+# Get current subscription ID
+$subscriptionId = (Get-AzContext).Subscription.Id
+
+$apiVersion = '2020-06-30'
+$token = Get-AzAccessToken -ResourceUrl https://search.azure.com -AsSecureString | Select-Object -ExpandProperty Token
+$uri = "https://$BaseName.search.windows.net"
+
+$indexDefinition = @{
+  name = "products"
+  fields = @(
```

Comment:
Sadly, many of these fields options default to true in the API, so we'll need to explicitly set the following options in all fields, either to true if explicitly true below, or to false if not mentioned: searchable, filterable, facetable, sortable.

Reply:
I'll default to false in a loop if not defined